viamonvif/discovery.go

```diff
@@ -356,7 +356,7 @@ func getRTSPStreamURLs(deviceInstance OnvifDevice, username, password string, lo
 		getStreamURI := media.GetStreamUri{
 			StreamSetup: onvifxsd.StreamSetup{
 				Stream:    onvifxsd.StreamType(streamTypeRTPUnicast),
-				Transport: onvifxsd.Transport{Protocol: "rtsp"},
+				Transport: onvifxsd.Transport{Protocol: "RTSP"},
```

Comment:
If this is actually ready for review - I have not seen a regression in-camera detection from running the discovery script on my machine - and we are detecting the Raymarine using this difference.